release/staging-resources-cdk/lib/StagingResourcesStack.ts

```diff
@@ -16,5 +23,51 @@ export class StagingResourcesStack extends Stack {
     this.dataPrepperEcrRepository = new CfnPublicRepository(this, 'DataPrepperECRRepository', {
       repositoryName: 'data-prepper'
     });
+
+    const archivesBucketName: string = scope.node.tryGetContext('archivesBucketName');
+
+    const archivesBucket = Bucket.fromBucketName(this, 'StagingBucket', archivesBucketName);
+    const originAccessIdentity = new OriginAccessIdentity(this, 'StagingOriginAccessIdentity');
+
+    const cloudFrontDistribution = new CloudFrontWebDistribution(this, 'StagingCloudFrontDistribution', {
+      originConfigs: [
+        {
+          s3OriginSource: {
+            s3BucketSource: archivesBucket,
+            originAccessIdentity: originAccessIdentity
+          },
+          behaviors: [
+            {
+              isDefaultBehavior: true
+            }
+          ]
+        }
+      ]
+    });
+
+    const policyStatement = new PolicyStatement({
+      actions: [
+        's3:GetObject'
+      ],
+      resources: [
+        archivesBucket.arnForObjects('*')
```

Comment:
Should this be restrictive ?

Reply:
This generates an ARN that looks like: `arn:aws:s3:::my-bucket-name/*`.

The entire S3 bucket is used for archives. The structure will look like the following:

```
/
  {version}/
    {buildNumber}/
      archive/
      maven/
```

So we do not know the exact key path to start.